docs/source/use_cases/discover_public_data/pgp_public_data.rst

```diff
@@ -28,7 +28,7 @@ This dataset comprises roughly 180 Complete Genomics genomes.  See the `Personal
 Google Cloud Platform data locations
 ------------------------------------
 
-* Google Cloud Storage folder `gs://pgp-harvard-data-public <https://console.developers.google.com/storage/pgp-harvard-data-public>`_
+* Google Cloud Storage folder `gs://pgp-harvard-data-public <https://console.cloud.google.com/storage/pgp-harvard-data-public>`_
 * Google Genomics Dataset ID `9170389916365079788 <https://developers.google.com/apis-explorer/#p/genomics/v1/genomics.datasets.get?datasetId=9170389916365079788>`_
```

Comment:
same thing as above - this one points to `v1` instead of `v1beta2` - is that okay?